app/assets/javascripts/rejection.js

```diff
@@ -0,0 +1,17 @@
+$(function() {
```

Comment:
No it is not, just thougt bootstrap has already a nice solution, this is why i mention it, could be a working solution